exercises/practice/largest-series-product/largest_series_product.h

```diff
@@ -0,0 +1,8 @@
+#if !defined(LARGEST_SERIES_PRODUCT_H)
+#define LARGEST_SERIES_PRODUCT_H
+
+namespace largest_series_product {
+
+} // largest_series_product
```

Comment:
I'd suggest `// namespace largest_series_product` instead of `// largest_series_product` (same as above).